assets/i18n/active.ja.json

```diff
@@ -0,0 +1,122 @@
+{
+  "command.default.no": {
+    "hash": "sha1-816c52fd2bdd94a63cd0944823a6c0aa9384c103",
+    "other": "いいえ"
+  },
+  "command.default.yes": {
+    "hash": "sha1-5397e0583f14f6c88de06b1ef28f460a1fb5b0ae",
+    "other": "はい"
+  },
+  "command.error.generic": {
+    "hash": "sha1-73c841bf1e817ec2ab09b17eb0b43ec61bf1eda9",
+    "other": "問題が発生しました。再度ためしてみてください。"
+  },
+  "command.error.invalidInput": {
+    "hash": "sha1-6ef073c836a398485967d7c10a085b611d272ddc",
+    "other": "無効な入力: {{.Error}}"
+  },
+  "command.error.invalidNumberOfOptions": {
+    "hash": "sha1-31287872531872ed11e9ae69975afc1cd2ee09ae",
+    "other": "一つも回答を入力しないか、もしくは二つ以上の回答を入力してください。"
+  },
+  "command.help.text.options": {
+    "hash": "sha1-9576f41168dfc7418562fad8a5e82f309014c344",
+    "other": "`/{{.Trigger}} \\\"Question\\\" \\\"Answer 1\\\" \\\"Answer 2\\\" \\\"Answer 3\\\"`のように入力することで、回答をカスタマイズすることができます。"
+  },
+  "command.help.text.pollSetting.anonymous": {
+    "hash": "sha1-fb98b7ba3c20091f6ea79a47ee43122d14e3f50a",
+    "other": "誰がどこに回答したのか表示しない"
+  },
+  "command.help.text.pollSetting.introduction": {
+    "hash": "sha1-2484e1a47494f152cf4e3d96bd173da1a779ac30",
+    "other": "投票に関する設定を行うことができます。例: `/{{.Trigger}} \\\"Question\\\" \\\"Answer 1\\\" \\\"Answer 2\\\" \\\"Answer 3\\\" --progress --anonymous`. 利用可能な設定:"
+  },
+  "command.help.text.pollSetting.progress": {
+    "hash": "sha1-9b8deba20f6a2fc6221841de9db19c903d28f97f",
+    "other": "投票期間中、それぞれの回答に集まった投票を表示する"
+  },
+  "command.help.text.pollSetting.public-add-option": {
+    "hash": "sha1-97f6e52cd7135ca4c73b79790a946ead7513cd33",
+    "other": "全てのユーザーが回答を追加できるようにする"
+  },
+  "command.help.text.simple": {
+    "hash": "sha1-3e9e692901434c94972562861477a71bf6dda12e",
+    "other": "\\\"{{.Yes}}\\\" と \\\"{{.No}}\\\" を回答とする投票を作成するには `/{{.Trigger}} \\\"Question\\\"` と入力してください。"
+  },
+  "dialog.addOption.element.displayName": {
+    "hash": "sha1-e31d972229de381b461860014dd710856361917a",
+    "other": "回答"
+  },
+  "dialog.addOption.submitLabel": {
+    "hash": "sha1-61cc55aa0453184734c3fa0b621eda6fa874bd83",
+    "other": "追加"
+  },
+  "dialog.addOption.title": {
+    "hash": "sha1-05448f4011e4f488f6cef7cfaac5e2364528b304",
+    "other": "回答を追加する"
+  },
+  "poll.button.addOption": {
+    "hash": "sha1-05448f4011e4f488f6cef7cfaac5e2364528b304",
+    "other": "回答を追加する"
+  },
+  "poll.button.deletePoll": {
+    "hash": "sha1-390ff4fdd0cfef6ef35fbf067a605aa3ea87ecca",
+    "other": "回答を削除する"
+  },
+  "poll.button.endPoll": {
+    "hash": "sha1-60ddde386bd7190fa3da2ba13f97e3caa625d3e6",
+    "other": "投票を終了する"
+  },
+  "poll.endPost.answer.heading": {
+    "hash": "sha1-46743e9dbbeec0c43d16fe387735cf95c2e27abb",
+    "other": "{{.Answer}} (投票数 {{.Count}})"
```

Comment:
Just to double check: There is no difference between the number of votes in the wording?

Reply:
Yes, Japanese has lesser cases for using plural than English, so I think this message would not be wrong.
(FYI: http://selftaughtjapanese.com/2014/02/05/plurality-in-japanese/)

Thanks for taking look deeply 👍